### PR TITLE
Ship 1.1.0

### PR DESCRIPTION
**Pull request by aanand**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,21 +1,8 @@
 Change log
 ==========
 
-1.1.0-rc2 (2015-01-29)
```
Comment (thaJeztah) [inline CHANGES.md]: Is there still a full history available somewhere after this has been removed? It's sometimes handy to be able to check when/in which version something was fixed/introduced.

Comment (aanand) [inline CHANGES.md]: We weren't planning on it, but the git tags will stick around, so that'll help when narrowing things down.

**Comment by bfirsh** [comment]
LGTM

**Comment by marksteve** [comment]
:tada:

**Pull request merged**